gulpfile.js

```diff
@@ -1,14 +1,11 @@
 var gulp = require('gulp');
 var jshint = require('gulp-jshint');
 
-var lintSrc = [
-  './lib/**/*.js',
-  './test/**/*.js',
-  './index.js'
-];
+var lintSrc = ['./lib/**/*.js', './test/**/*.js'];
 
-gulp.task('lint', function () {
-  return gulp.src(lintSrc)
-    .pipe(jshint({esnext:true}))
+gulp.task('lint', function() {
+  return gulp
+    .src(lintSrc)
+    .pipe(jshint({ esversion: 6 }))

```

Comment:
Why are we doing this?

Reply:
Warning This option has been deprecated and will be removed in the next major release of JSHint. Use esversion: 6 instead.